Formula/helix.rb

```diff
@@ -0,0 +1,22 @@
+class Helix < Formula
+  desc "Post-modern modal text editor"
+  homepage "https://helix-editor.com"
+  url "https://github.com/helix-editor/helix.git",
+       tag:      "v0.6.0",
+       revision: "ac1b7d8d0a608f47edfee2872d414e94fd26cc31"
```

Comment:
Do we need a git clone? Can we use a tarball instead?

Reply:
I think so. There are 55 submodules need to be fetched for building the binary, which are not available in the tarball.